news/2022-02-15-beatmap-spotlights-season-5-winter-2022.md

```diff
@@ -0,0 +1,162 @@
+
+### More transparency
+
+There's been a lot of voices that nobody really knows who's the member and what's going on in the Spotlighs Team. We'd like to face this issue, and make it so everything is a little bit more clear to the outsiders. Each season announcement will now include *[Inside the spotlights team](inside-the-team)* section, listing the most recent team changes. We will also credit respective curators, resposnible for suggesting the maps, in the [picks lists](playlists). There's more to come in that regard, so stay tuned for the next seasons!
+
+## Medals and beatmap packs
+
+Many people stated that they'd like to enjoy some kind of spotlights experience on stable, like it used to be in the past. To make it somehow possible, we will now release beatmap packs along with medal set for each gamemode once all of the playlists are released (in 4 weeks). We also have plans for releasing medal packs for previous Spotlights seasons. Have fun jumping on grindset to catch them all!
+
+## <a id="inside-the-team"></a> Inside the spotlights team
+
+In this section we will cover member changes and important happenings inside the Spotlights Team.
+
+**Curator additions**
+- **osu!taiko**: Hivie, radar
+- **osu!catch**: Rocma
+- **osu!mania**: \[ Decku \], Kibitz
+
+**Curator resignments**
+- **osu!**: Icekalt, xbopost
+- **osu!taiko**: Charlotte
+- **osu!catch**: Deif, JBHyperion, Spectator
+- **osu!mania**: Raveille
+
+In addition to that, pishifat will leave the management. After a long service, Hivie will replace him as a helping hand in making the things work as intended.
+
+For the full Spotlights Team members list, check out the [Beatmap Spotlights wiki article](/wiki/Beatmap_Spotlights).
+
+## <a id="how-to-participate"></a> How to participate in Beatmap Spotlights League?
+
+To participate in the Beatmap Spotlights League, you need to download the latest version of osu!lazer client from [GitHub releases page](https://github.com/ppy/osu/releases/latest). Once you're done installing and logging in to your account, head on over to the playlists tab in the play section and find the Spotlights lobbies as denoted by a little pink bar. You can also filter the lobbies to spotlights only, by selecting so in the top right corner.
+
+## Season schedule and leaderboards
+
+| Event | Time |
+| :-- | :-- |
+| Season Start | Feb 15th, 2022 |
+| Timeshift Lobby: Playlist A | Feb 15th - Feb 29th, 2022 |
+| Timeshift Lobby: Playlist B | Feb 29th - Mar 15th, 2022 |
+| Timeshift Lobby: Playlist C | Mar 15th - Mar 29th, 2022 |
+| Timeshift Lobby: Playlist A | Mar 29th - Apr 5th, 2022 |
+| Timeshift Lobby: Playlist B | Apr 5th - Apr 12th, 2022 |
+| Timeshift Lobby: Playlist C | Apr 12th - Apr 19th, 2022 |
+| Season Conclusion | Apr 19th, 2022 |

```

Comment:
we usually use sentence case for tables (see tournament articles)

also would rather make the date format more concise

```suggestion
| Season start | Feb 15, 2022 |
| Timeshift lobby: Playlist A | Feb 15 - Feb 29, 2022 |
| Timeshift lobby: Playlist B | Feb 29 - Mar 15, 2022 |
| Timeshift lobby: Playlist C | Mar 15 - Mar 29, 2022 |
| Timeshift lobby: Playlist A | Mar 29 - Apr 5, 2022 |
| Timeshift lobby: Playlist B | Apr 5 - Apr 12, 2022 |
| Timeshift lobby: Playlist C | Apr 12 - Apr 19, 2022 |
| Season conclusion | Apr 19, 2022 |
```